package.json

```diff
@@ -57,7 +57,8 @@
     "@web3auth/base-provider": "^7.0.1",
     "bn.js": "^5.2.1",
     "bowser": "^2.11.0",
-    "elliptic": "^6.5.4"
+    "elliptic": "^6.5.4",
+    "ts-custom-error": "^3.3.1"
```

Comment:
can we just copy the required files from this lib, rather thn downloading. I checked we just need 1 class and 2 utility functions from this lib.